clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java

```diff
@@ -132,16 +132,25 @@ public void clusteredCachePutGet() throws Exception {
       Cache<Long, String> cache = cacheManager.createCache("clustered-cache", config);
       assertThat(cache.get(1L), is(nullValue()));
       cache.put(1L, "The one");
+      assertThat(cache.containsKey(2L), is(false));
```

Comment:
I do not like having asserts in code that used in docs. The callout should indicate the expected result and you may have asserts outside of what's included to make sure nothing breaks.